Sources/PackageListTool/Models/SwiftPackageIndexAPI.swift

```diff
@@ -33,4 +37,53 @@ struct SwiftPackageIndexAPI {
         let (data, _) = try await URLSession.shared.data(for: req)
         return try Self.decoder.decode(Package.self, from: data)
     }
+
+    func search(query: String, limit: Int) async throws -> [PackageId] {
+        var urlComponents = URLComponents(string: "\(baseURL)/api/search")
+        urlComponents?.queryItems = [
+            URLQueryItem(name: "query", value: query),
+            URLQueryItem(name: "page", value: "1"),
+            URLQueryItem(name: "pageSize", value: "\(limit)"),
```

Comment:
The `pageSize` parameter does not harm but will be ignored until we deploy https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/pull/2560